src/Mod/TechDraw/Gui/CommandExtensionDims.cpp

```diff
@@ -1864,7 +1894,7 @@ void execCreateVertChamferDimension(Gui::Command* cmd) {
     TechDraw::DrawViewPart* objFeat;
     if (!_checkSelAndObj(cmd, selection, objFeat, "TechDraw Create Vertical Chamfer Dimension"))
         return;
-    Gui::Command::openCommand(QT_TRANSLATE_NOOP("Command", "Vert Chamfer Dim"));
+    Gui::Command::openCommand(QT_TRANSLATE_NOOP("Command", "Create Vert Chamfer Dim"));
```

Comment:
Wouldn't it make sense to spell out `Vertical` fully here?

Reply:
AFAIK this text is used to identify commits. The user will only be able to see this when using the dropdown arrows next to Undo and Redo. In this sense it is not very important.

My reason for adding "Create" is that the command is called `TechDraw_ExtensionCreateVertChamferDimension`.

I have mainly focused on menu texts and tooltips. This was already an involved process that took a lot more time than anticipated. I was tempted to do more, I particularly don't like some of the command names, but you have to draw the line somewhere...

Reply:
Ok, fair enough